Add plugin: EUpload

Appleec opened the pull request:
# I am submitting a new Community Plugin

## Repo URL

<!--- Paste a link to your repo here for easy access -->
Link to my plugin: https://github.com/Appleec/e-obsidian-upload-plugin

## Release Checklist
- [ ] I have tested the plugin on
  - [x]  Windows
  - [x]  macOS
  - [ ]  Linux
  - [ ]  Android _(if applicable)_
  - [ ]  iOS _(if applicable)_
- [x] My GitHub release contains all required files (as individual files, not just in the source.zip / source.tar.gz)
  - [x] `main.js`
  - [x] `manifest.json`
  - [x] `styles.css` _(optional)_
- [x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the developer policies at https://docs.obsidian.md/Developer+policies, and have assessed my plugins's adherence to these policies.
- [x] I have read the tips in https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

joethei commented:
What sets your plugin appart from some of the other image upload plugins?
There are multiple that already support Lskypro / GitHub

Remove the `styles.css` file from your release, since it's empty it's just unnecessary bloat that Obsidian would download.

[containerEl.createEl("h1", { text: "上传设置" });](https://github.com/Appleec/e-obsidian-upload-plugin/blob/be8f6ed50747e468feec346da6958918e775ce02/src/settings-tab.ts#L35)
[Don't add a top-level heading in the settings tab, such as "General", "Settings", or the name of your plugin.](https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines#Only+use+headings+under+settings+if+you+have+more+than+one+section.)

[const adapter = (this.app.vault.adapter instanceof FileSystemAdapter) ?](https://github.com/Appleec/e-obsidian-upload-plugin/blob/be8f6ed50747e468feec346da6958918e775ce02/src/utils/helper.ts#L129-L131)
The adapter on mobile does not have a `getBasePath` function, so this code would crash,

Appleec commented:
Hi, @joethei :

First of all, thank you for your reply!

The following modifications have been made to your question:
- Removed the empty `styles.css` file in the latest release as requested.
- Modified the top-level title in the settings tab as requested.
- Mobile is not supported yet, but a simple logical judgment is made on `adapter.getBasePath` to prevent exceptions.

Looking forward to your reply.

Appleec commented:
Hi, @joethei  :

First of all, thank you for your reply!

The following modifications have been made to your question:

- Compared with similar plug-ins, this plug-in configuration is customizable and more flexible, and more support and operation space will be provided in the future.
- Removed the empty styles.css file in the latest release as requested.
- Modified the top-level title in the settings tab as requested.
- Mobile is not supported yet, but a simple logical judgment is made on adapter.getBasePath to prevent exceptions.

Looking forward to your reply.

Appleec commented:
Hi, @joethei :

First of all, thank you for your reply!

Make the following changes:

- Temporarily removed the [top-level heading ](https://github.com/Appleec/e-obsidian-upload-plugin/blob/eeb082060c00df9c30e5dd7f15a4f53d2b4360d0/src/settings-tab.ts#L35)because I didn't think of a good title.
- Modified [adapter.getBasePath](https://github.com/Appleec/e-obsidian-upload-plugin/blob/eeb082060c00df9c30e5dd7f15a4f53d2b4360d0/src/utils/helper.ts#L127) according to your suggestion to prevent abnormalities on mobile.

Looking forward to your reply.

joethei commented:
There should not be a top level heading, see the link.

Appleec commented:
Thanks for your suggestion, I have made the changes. If you have any questions, please let me know.
